### PR TITLE
Cleaning the code

### PR DESCRIPTION
**Pull request by Domel303**
Cleaning the code. Creating new classes and methods instead of having all logic in one class. Refactoring names of variables and methods.

### CODE REVIEW

File: INPTPZ1/Fractals/NewtonFractals.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                coordinates = coordinates.Subtract(difference);
+
+                if (InNewtonsLimit(difference))
+                    i--;
+                
+                iteractionNumber++;
+            }
+            return coordinates;
+        }
+        private int findRoot(List<Complex> roots, Complex coordinates)
```
Comment (rodi0878) [inline INPTPZ1/Fractals/NewtonFractals.cs]: method naming convention is not respected

File: INPTPZ1/Fractals/NewtonFractals.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        private static readonly int IterationLimit = 30;
+        private static readonly double NewtonsFractalLimit = 0.5;
+        private static readonly int RGBColorValue = 255;
+        private static readonly Color[] colors =
+        {
+            Color.Red, Color.Blue, Color.Green,
+            Color.Yellow, Color.Orange, Color.Fuchsia,
+            Color.Gold, Color.Cyan, Color.Magenta
+        };
+        private static Polynomial StartFunction()
```
Comment (rodi0878) [inline INPTPZ1/Fractals/NewtonFractals.cs]: methods should have a verb in their name - also repeats for many other methods below this

File: INPTPZ1/Fractals/NewtonFractals.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+using System.Drawing;
+
+namespace INPTPZ1.Fractals
+{
+    public class NewtonFractals
+    {
+        private static readonly int IterationLimit = 30;
+        private static readonly double NewtonsFractalLimit = 0.5;
+        private static readonly int RGBColorValue = 255;
+        private static readonly Color[] colors =
```
Comment (rodi0878) [inline INPTPZ1/Fractals/NewtonFractals.cs]: why "colors" when you have "IterationLimit" and others...

File: INPTPZ1/INPTPZ1.csproj.user [hunk 4]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (rodi0878) [inline INPTPZ1/INPTPZ1.csproj.user]: user configuration files must not be added to repository

**Review by rodi0878 — CHANGES_REQUESTED** [review]
Code contains naming errors please fix them before merge.

**Comment by Domel303** [comment]
@rodi0878 Errors were fixed.

**Comment by rodi0878** [comment]
Accepted

**Pull request closed without merging**